src/engine/uct/node/mod.rs

```diff
@@ -140,11 +140,9 @@ impl Node {
         }
         best
     }
-
-    pub fn all_losses(&self) -> bool {
-        self.children
-            .iter()
-            .all(|n| n.wins == 0)
+    
+    pub fn mostly_losses(&self) -> bool {
+        self.win_ratio() < 0.01
```

Comment:
it shouldn't be a constant per se, it should depend on the number of playouts, but I'd have to think of how to calculate it